code/Bids_Exploration.ipynb

```diff
@@ -681,86 +699,97 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
-   "id": "db27c3f4",
+   "execution_count": 27,
+   "id": "aeb246d2",
    "metadata": {},
    "outputs": [],
    "source": [
-    "example_filename = os.path.join(data_path, 'sub-01_ses-anatomy_T1w.nii.gz')"
+    "anat = os.path.join(data_path,'sub-01_ses-anatomy_T1w.nii.gz')"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 25,
-   "id": "4d2a4b8c",
+   "execution_count": 35,
+   "id": "c062ea6d",
    "metadata": {},
-   "outputs": [
-    {
-     "ename": "SyntaxError",
-     "evalue": "invalid token (2178944879.py, line 1)",
-     "output_type": "error",
-     "traceback": [
-      "\u001b[0;36m  File \u001b[0;32m\"/tmp/ipykernel_727/2178944879.py\"\u001b[0;36m, line \u001b[0;32m1\u001b[0m\n\u001b[0;31m    anat = sub-01_ses-anatomy_T1w.nii.gz\u001b[0m\n\u001b[0m                 ^\u001b[0m\n\u001b[0;31mSyntaxError\u001b[0m\u001b[0;31m:\u001b[0m invalid token\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
-    "anat = sub-01_ses-anatomy_T1w.nii.gz"
+    "fmri_path = '/home/jpauli/ds001506/sub-01/ses-imagery01/anat'"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 28,
-   "id": "0aaf43d0",
+   "execution_count": 36,
+   "id": "352640f6",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import nibabel as nib"
+    "fmri_img = os.path.join(fmri_path,'sub-01_ses-imagery01_inplaneT2.nii.gz')"
```

Comment:
the path points to an inplane `T2w` image within the `anat` folder which indicates that you are not loading an `fMRI` image. please have a look at the BIDS folder structure and respective terms again.